### PR TITLE
PAL-695 Jenkins Build Times

### PR DESCRIPTION
**Pull request by dbz120896**
Just added timestamps and the sonarqube section to name the analysis by the branch name if it's a PR.

### CODE REVIEW

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]
Whats with all the whitespace changes? 
The SonarQube stage needs to be replicated in other repositories too

File: Jenkinsfile [hunk 1]
```diff
@@ -26,274 +26,280 @@
...
+        imagePullPolicy: IfNotPresent
+        command:
+        - sleep
+        args:
+        - 99d
+        env:
+          - name: DOCKER_HOST
+            value: tcp://localhost:2375
+        resources:
+          requests:
```
Comment (ac74475) [inline Jenkinsfile]:
```suggestion
          requests:
            cpu: "2"
```

Comment (dbz120896) [reply Jenkinsfile]: Aha! That's where :)

Comment (dbz120896) [reply Jenkinsfile]: I take it this would be in all the other repositories as well?

**Comment by dbz120896** [comment]
> Whats with all the whitespace changes?
> The SonarQube stage needs to be replicated in other repositories too

I've added the timestamps plugin. Everything inside that is now indented. Maybe that did it.

**Review by pd104923 — APPROVED** [review]

**Review by developer01189998819991197253 — APPROVED** [review]

**Pull request merged**